### PR TITLE
fix(CTS): cts supports the cross account feature

### PR DESCRIPTION
**Pull request by wanglong3710**
cts supports the cross account feature

**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed
make testacc TEST='./huaweicloud/services/acceptance/cts/' TESTARGS='-run TestAccCTSTracker_basic' 
```
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/cts/ -v -run TestAccCTSTracker_basic -timeout 360m -parallel 4 
=== RUN   TestAccCTSTracker_basic 
=== PAUSE TestAccCTSTracker_basic
=== CONT  TestAccCTSTracker_basic
--- PASS: TestAccCTSTracker_basic (71.59s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cts       71.628s
```

### CODE REVIEW

**Comment by deer-hang** [comment]
please run your unit test.

File: docs/resources/cts_tracker.md [hunk 1]
```diff
@@ -12,9 +12,10 @@ Manages CTS **system** tracker resource within HuaweiCloud.
 variable "bucket_name" {}
 
 resource "huaweicloud_cts_tracker" "tracker" {
-  bucket_name = var.bucket_name
-  file_prefix = "cts"
-  lts_enabled = true
+  bucket_name          = var.bucket_name
+  file_prefix          = "cts"
+  lts_enabled          = true
+  organization_enabled = true
```
Comment (deer-hang) [inline docs/resources/cts_tracker.md]: remove this line

File: docs/resources/cts_tracker.md [hunk 2]
```diff
@@ -33,6 +33,11 @@ The following arguments are supported:
 
 * `lts_enabled` - (Optional, Bool) Specifies whether trace analysis is enabled.
 
+* `organization_enabled` - (Optional, Bool) Specifies whether to apply the tracker configuration to the organization.
+  This parameter is valid only for the management tracker. If the value is set to true, the audit logs of all members
+  in the ORG organization in the current region will be transferred to the OBS bucket or LTS log stream configured for
+  the management tracker. However, audit logs of other members cannot be viewed on the Trace List page.
```
Comment (ShiChangkuo) [inline docs/resources/cts_tracker.md]: the type of tracker is already **system**

```suggestion
  If the value is set to **true**, the audit logs of all members
  in the organization in the current region will be transferred to the OBS bucket or LTS log stream configured for
  the management tracker.
```

**Pull request merged**